src/bar_ros/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.5.0)
+project(bar_ros)
+
+find_package(catkin QUIET)
+find_package(ament_cmake QUIET)
```

Comment:
Please note that both could be found in situations where both ROS 1 and ROS 2 have been `source`d (fi when the `ros1bridge` is being used).

Reply:
Is this something you would do when building or only when running things?

Reply:
I am assuming this should not be done when building so I could added a check if both are found it would error.

Reply:
> Is this something you would do when building or only when running things?

Both. For "custom messages" the bridge needs to be rebuilt. That would be something users will want to do I believe in many cases.

> I am assuming this should not be done when building so I could added a check if both are found it would error.

If with "this" you are referring to "`source`ing both" then it would be done both at runtime and at build time.

Reply:
If this is the case then will there be package name conflicts between ros1 and ros2 packages with the same name?